documentation/src/main/asciidoc/userguide/appendices/Annotations.adoc

```diff
@@ -731,7 +731,7 @@ See the <<chapters/domain/basic_types.adoc#mapping-column-read-and-write-example
 [[annotations-hibernate-columntransformers]]
 ==== `@ColumnTransformers`
 
-The https://docs.jboss.org/hibernate/orm/{majorMinorVersion}/javadocs/org/hibernate/annotations/ColumnTransformers.html[`@ColumnTransformers`] annotation iis used to group multiple <<annotations-hibernate-columntransformer>> annotations.
+The https://docs.jboss.org/hibernate/orm/{majorMinorVersion}/javadocs/org/hibernate/annotations/ColumnTransformers.html[`@ColumnTransformers`] annotation is used to group multiple <<annotations-hibernate-columntransformer>> annotations.
 
```

Comment:
The above typo correcting is unrelated to this PR, but it seems harmless to include such small fix for it is of such a small scope.